libs/entitlement/bridge/build.gradle

```diff
@@ -16,10 +19,14 @@ configurations {
   }
 }
 
+tasks.named('jar').configure {
+  from sourceSets.main23.output
+}
+
 artifacts {
   bridgeJar(jar)
```

Comment:
you declare artifacts for external use but in the consuming project we only refer to default or the java23 configuration it seems

Reply:
Ah, I see. I guess this is leftover from a past layout. I've removed it.